### DISCUSSION TITLE
Site map for human visitors?

### DISCUSSION THREAD

**Discussion by Wysardry** [getnikola/nikola#3693] [Q&A] [answered]
How would I go about adding a site map for human visitors that will update automatically whenever a post/page is added or modified?

Would I need to create a completely new template for that?

**Comment by sukiletxe** [comment, marked as answer]
See the [post list directive / shortcode](https://getnikola.com/handbook.html#toc-entry-88).

**Comment by Wysardry** [comment]
Thanks. I didn't notice the post_types filter when I skimmed the docs, so I thought it was only meant for listing posts.
